pkg/nimble/Makefile.include

```diff
@@ -93,15 +93,27 @@ ifneq (,$(filter nimble_netif,$(USEMODULE)))
 
   # configure NimBLE's internals
   NIMBLE_MAX_CONN ?= 3
-  CFLAGS += -DMYNEWT_VAL_MSYS_1_BLOCK_SIZE=264
   CFLAGS += -DMYNEWT_VAL_BLE_L2CAP_COC_MAX_NUM=$(NIMBLE_MAX_CONN)
   CFLAGS += -DMYNEWT_VAL_BLE_MAX_CONNECTIONS=$(NIMBLE_MAX_CONN)
-  # NimBLEs internal buffer need to hold one IPv6 MTU per connection
-  # for the internal MTU of 256 byte, we need 10 mbufs per connection...
+
+  # the maximum fragment size that we can receive. For maximum efficiency this
+  # should be equal to the maximum configured link layer packet size.
+  # WARNING: this value MUST never be larger than MYNEWT_VAL_BLE_LL_MAX_PKT_SIZE
+  CFLAGS += -DMYNEWT_VAL_BLE_L2CAP_COC_MPS=251
+
+  # in order to fit a 251 byte COC data segment into a single mbuf buffer, the
+  # used block size must be at least 297 byte (251 data + 48 overhead)
+  CFLAGS += -DMYNEWT_VAL_MSYS_1_BLOCK_SIZE=299
```

Comment:
Can't this be defined in function of `MYNEWT_VAL_BLE_L2CAP_COC_MPS`?

Reply:
Yes, we could do that. Will fix.